src/app/modules/bounce/bounce.util.ts

```diff
@@ -13,7 +9,7 @@ export const extractHeader = (
   body: IEmailNotification,
   header: string,
 ): string => {
-  return get(body, 'mail.headers').find(
+  return body.mail.headers.find(
```

Comment:
what's wrong with using lodash `get`? I think the fix here is that the default should be set for `get` to be `[]`?

Reply:
Yeah so the issue is that once you use `_.get`, you lose the typing - everything gets converted to `any`. But you're right that there is still a danger because we are parsing a POST notification from an external source, which is why all the processing is wrapped in a try-catch in the controller (at least until we implement FP).

Reply:
In fact once TypeScript strict mode turns on, we can't use _.get anymore